src/cursor.ts

```diff
@@ -224,4 +235,8 @@ export default class Cursor {
 
 		return null;
 	}
+
+	private static posOfFirstNonBlankChar(line: number): number {
```

Comment:
In the latest for `master`, the is a similar implementation for this. Would you be able to reuse the `getNextWordPosition` or `getPreviousWordPosition` functions?